met-api/src/met_api/models/engagement.py

```diff
@@ -63,6 +64,31 @@ def get_engagements_paginated(cls, pagination_options: PaginationOptions, search
         if search_text:
             query = query.filter(Engagement.name.ilike('%' + search_text + '%'))
 
+        if advanced_search_params:
+            if advanced_search_params['createdfromdate']:
```

Comment:
you can use walrus := operator to simplify this...

Reply:
Please add a meaningful separator in the variable names everywhere ;like instead of createdfromdate, i think putting underscores like created_from_date will be helpful..
apply to all variables

Reply:
```
if (published_from_date := advanced_search_params['publishedfromdate']):
    query = query.filter(Engagement.published_date >= published_from_date)
```

this is how the walrus operator gonna look like

Reply:
Updated. Thanks.